packages/wdio-mocha-framework/src/constants.ts

```diff
@@ -1,9 +1,15 @@
 export const INTERFACES = {
-    bdd: ['it', 'before', 'beforeEach', 'after', 'afterEach'],
+    bdd: ['it', 'specify', 'before', 'beforeEach', 'after', 'afterEach'],
     tdd: ['test', 'suiteSetup', 'setup', 'suiteTeardown', 'teardown'],
     qunit: ['test', 'before', 'beforeEach', 'after', 'afterEach']
 } as const
 
+export const TEST_INTERFACES = {
```

Comment:
Why are these added? Having the INTERFACES seems like it already covers everything we need.

Reply:
I solved this in a similar way it is being done for jasmine. https://github.com/webdriverio/webdriverio/blob/main/packages/wdio-jasmine-framework/src/index.ts#L20 - it has interfaces and test interfaces are two different constants..